setup.py

```diff
@@ -14,12 +14,15 @@
     "colorama==0.4.4",  # Colorizes terminal output
     "colorlog==5.0.1",  # Adds color to logs
     "concurrent-log-handler==0.9.19",  # Concurrently log and rotate logs
-    "cryptography==3.4.7",  # Python cryptography library for TLS - keyring conflict
-    "fasteners==0.16.3",  # For interprocess file locking
-    "keyring==23.0.1",  # Store keys in MacOS Keychain, Windows Credential Locker
-    "keyrings.cryptfile==1.3.4",  # Secure storage for keys on Linux (Will be replaced)
-    #  "keyrings.cryptfile==1.3.8",  # Secure storage for keys on Linux (Will be replaced)
-    #  See https://github.com/frispete/keyrings.cryptfile/issues/15
+    # Python cryptography library for TLS - keyring conflict
+    "cryptography==3.4.7",
+    # Store keys in MacOS Keychain, Windows Credential Locker
+    "keyring==23.0.1",
+    # Secure storage for keys on Linux (Will be replaced)
+    "keyrings.cryptfile==1.3.4",
+    # "keyrings.cryptfile==1.3.8"
+    # Secure storage for keys on Linux (Will be replaced)
+    # See https://github.com/frispete/keyrings.cryptfile/issues/15
```

Comment:
where these changes deliberate or a rebase-issue?

Reply:
Sorry, my original intention was just to modify the coding error problem, and then found that the reading of the file is also a bit of a problem, so I added a function containing the with to automatically manage the file. But I found that previous submissions had not passed, and one contributor told me that I needed to use black and flake8 to check the file format and code style, so I modified the previous comments.